man/Makefile-server.am

```diff
@@ -5,8 +5,12 @@ dist_man_MANS += \
 	mount.ceph.8 \
 	ceph-create-keys.8 \
 	ceph-rest-api.8 \
-	ceph-debugpack.8 \
+	ceph-debugpack.8
+
+if WITH_SELINUX
```

Comment:
verified that ceph_selinux.8 is only included with %if 0%{with selinux} in ceph.spec.in, therefore conditionally building it is not going to cause a failure because the package cannot find it.